src/libpriv/rpmostree-postprocess.c

```diff
@@ -512,34 +512,68 @@ rpmostree_prepare_rootfs_get_sepolicy (int            dfd,
   return ret;
 }
 
-static gboolean
-replace_nsswitch (int            dfd,
-                  GCancellable  *cancellable,
-                  GError       **error)
+char *
+rpmostree_postprocess_replace_nsswitch_string (const char *buf,
+                                               GError    **error)
 {
-  g_autofree char *nsswitch_contents = NULL;
-  g_autofree char *new_nsswitch_contents = NULL;
+  gboolean is_passwd;
+  gboolean is_group;
+
+  is_passwd = g_str_has_prefix (buf, "passwd:");
+  is_group = g_str_has_prefix (buf, "group:");
```

Comment:
Wait, I'm confused. We're passing in the whole contents of `nsswitch.conf` right? So shouldn't we also have to split the lines and find `passwd:` and `group:`?